app/client/lib/uploads.ts

```diff
@@ -31,6 +31,9 @@ export interface SelectFileOptions extends UploadOptions {
                           // e.g. [".jpg", ".png"]
 }
 
+// This list coincides with the extensions defined in core/plugins/manifest.yml
+export const IMPORTABLE_TABLE_EXTENSIONS = [".xlsx", ".json", ".csv", ".tsv", "dsv"];
```

Comment:
The relationship between `IMPORTABLE_EXTENSIONS` and `IMPORTABLE_TABLE_EXTENSIONS` is a bit confusing. The word TABLE suggests one of them might be for single table imports, but that isn't the case for .xlsx and .json

Maybe the distinction is more like `IMPORTABLE_AS_DOC_EXTENSIONS` and `IMPORTABLE_WITHIN_DOC_EXTENSIONS`?

Reply:
Okay, I've made that change.